model/version.go

```diff
@@ -118,6 +123,21 @@ func (self *Version) UpdateBuildVariants() error {
 	)
 }
 
+func (self *Version) UpdateActivation() error {
+	if utility.FromBoolPtr(self.Activated) {
+		return nil
+	}
+	self.Activated = utility.ToBoolPtr(true)
```

Comment:
We actually also have a utility.TruePtr() which I think reads better but it's moot